src/scenes/gameplay-scene.ts

```diff
@@ -436,6 +426,9 @@ export class GameplayScene {
 
   loadPuzzle = (isTimerEnded?) => {
     this.removeEventListeners();
+    
+    // this.feedbackTextEffects.clearParticle();
+
```

Comment:
If it is not needed, do not include it.

Reply:
@bernhardccodev thank you. i removed it. please check